compiler/rustc_parse/messages.ftl

```diff
@@ -864,6 +868,11 @@ parse_unmatched_angle_brackets = {$num_extra_brackets ->
            *[other] remove extra angle brackets
         }
 
+parse_unsafe_attr_outside_unsafe = unsafe attribute used without unsafe
```

Comment:
Discussion: maybe "unsafe attribute used without unsafe attribute block"? (that's the term used in the RFC reference-level explanation.